quattor/types/hardware.pan

```diff
@@ -186,6 +185,21 @@ type structure_hardware = {
     "console"      ? structure_console
     "sysloc"       ? structure_sysloc
     "nodename"     ? string
+<<<<<<< HEAD
+    # benchmarks is used to hold the performance benchmark for the machine 
+    # i.e. HEPSpec06 score
+    # this might be used to scale things such as the wall time
+    # example of using this might be : 
+    # variable CONDOR_WN_SCALING_FACTOR = value('/hardware/benchmark/HEPSpec06') 
+=======
+    #benchmarks is used to hold the performance benchmark for the machine 
+    #i.e. HEPSpec06 score
+    #this might be used to scale things such as the wall time
+    #example of using this might be : 
+    #variable CONDOR_WN_SCALING_FACTOR = value('/hardware/benchmark/HEPSpec06') 
+>>>>>>> origin/master
+    #       / ( 4 * get_num_of_cores()) ;
+    "benchmarks"    ? double{} 
```

Comment:
you like typos much? i know that adding site specific tests and tus naming can clutter the schema, but this is not ok imho.
we can ship a pretty empty `quattor/types/benchmarks.pan` and let the site decide/override?